script/installation/packages.sh

```diff
@@ -64,6 +64,8 @@ install_mac() {
   fi
   # Update Homebrew.
   brew update
+  brew upgrade
+  brew cleanup
```

Comment:
This seems like its fixing an unrelated problem.  Is it OK for these changes to wait on this PR?

Reply:
Nope, this wouldn't have gotten through CI without it. Needs to be triggered from time to time and this was faster than bugging Chad. Plan was to revert on the review commit.